test/jdk/java/net/InetAddress/OfLiteralTest.java

```diff
@@ -101,6 +105,33 @@ private static Stream<Arguments> validLiteralArguments() throws Exception {
         byte[] ipv6Ipv4MappedAddressExpBytes = new byte[]{
                 (byte) 129, (byte) 144, 52, 38};
 
+        // 87.0.0.1 address bytes
+        byte[] ipv4_87_0_0_1 = new byte[]{87, 0, 0, 1};
+
+        // 127.0.0.1 address bytes
+        byte[] ipv4_127_0_0_1 = new byte[]{127, 0, 0, 1};
+
+        // 17.99.141.27 address bytes
+        byte[] ipv4_17_99_141_27 = new byte[]{17, 99, (byte)141, 27};
+
+        // 127.8.0.1 address bytes
+        byte[] ipv4_127_8_0_1 = new byte[]{127, 8, 0, 1};
+
+        // 0.0.0.42 address bytes
+        byte[] ipv4_0_0_0_42 = new byte[]{0, 0, 0, 42};
+
+        // 0.0.0.34 address bytes
+        byte[] ipv4_0_0_0_34 = new byte[]{0, 0, 0, 34};
+
+        // 127.0.1.1 address bytes
```

Comment:
Update comment to match the code:
```suggestion
        // 127.0.1.2 address bytes
```